package/MDAnalysis/converters/OpenMMParser.py

```diff
@@ -105,16 +108,37 @@ def _mda_topology_from_omm_topology(self, omm_topology):
         -------
         top : MDAnalysis.core.topology.Topology
 
+
+        .. versionchanged:: 2.2.0
+           * The parser now works when element attribute is missing in some or
+             all the atoms.
+           * When none of the elements are present their atomtypes are guessed
+             using their names and their masses are then guessed using their
+             atomtypes.
+           * When partial elements are present, values from available elements
+             are used whereas the absent elements are assigned an empty string
+             with their atomtype set to 'X' and mass set to 0.0.
+           * For elements with invalid and unreal symbols, the symbol is used
+             as it is for atomtypes but an empty string is used for elements.
```

Comment:
Move these to a `Notes` section just above instead. Keep versionchanged brief and instead provide the information that's going to be useful going forward in the docstring.